airflow/gcp/operators/bigquery.py

```diff
@@ -259,6 +267,9 @@ class BigQueryGetDataOperator(BaseOperator):
     :type delegate_to: str
     :param location: The location used for the operation.
     :type location: str
+
+    :rtype: dict
+        Map containing the requested rows + metadata.
```

Comment:
That's the return value of the `execute` function, isn't it? `:rtype: dict` will show up in an IDE that `__init__` is returning a dict which is wrong. You also find the return value of `__init__` in line 290 (`-> None:`)